packages/vue/src/composables/useAuth.ts

```diff
@@ -1,43 +1,29 @@
 import { useActor } from '@xstate/vue';
-import { ref, reactive, Ref, watchEffect } from 'vue';
-import { getServiceFacade } from '@aws-amplify/ui';
+import { ref, reactive, Ref } from 'vue';
+import { AuthenticatorServiceFacade } from '@aws-amplify/ui';
 import { facade } from './useUtils';
 import { InterpretService } from '@/components';
 
 const service = ref() as Ref<InterpretService>;
-const useAuthenticatorValue = reactive({
-  ...facade,
-  send: '' as unknown,
-  state: '' as unknown,
-}) as any;
 
+/** @deprecated For internal use only */
 export const useAuth = (serv?: InterpretService) => {
```

Comment:
Eventually this will be completly removed, and we will no longer directly use `state` or `send`. And rely on the facades like we do for `React`

Reply:
Is this a public export?

Reply:
No internal, no public export.  Only `useAuthenticator` is public

Reply:
Does it need the "For internal use only" comment then?

Reply:
I can remove that, but leave the deprecated command, for developers internally

Reply:
[dedc4ed](https://github.com/aws-amplify/amplify-ui/pull/3469/commits/dedc4ed64daa51dca51aff763f6c324d86312c6c)